http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -965,6 +974,62 @@ class GKEInterpreter[F[_]](
 
     } yield ()
 
+  private[util] def installCromwellApp(helmAuthContext: AuthContext,
```

Comment:
would be nice to make these `installX` methods duplicate less...but I guess that's sth leo team can work on

Reply:
I agree that seeing all the duplication is painful. I've had positive experiences in the past with using a plugin architecture (it was in Python) to allow registration of "apps" without having to add custom code for each one.